CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ### 0.6.2 (Next)
 
-* Your contribution here.
+* Fix bug when using multiple trackers
```

Comment:
Describe the actual bug. Conform to other lines. Put back the contributing message.

Reply:
Will do. thanks.